torch/csrc/jit/mobile/backport_v5_to_v4.cpp

```diff
@@ -0,0 +1,77 @@
+#include <ATen/core/ivalue.h>
+//#include <caffe2/serialize/file_adapter.h>
+#include <caffe2/serialize/inline_container.h>
+#include <torch/csrc/jit/mobile/backport_factory.h>
+#include <torch/csrc/jit/mobile/model_compatibility.h>
+#include <torch/csrc/jit/mobile/module.h>
+#include <torch/csrc/jit/serialization/pickler.h>
+
+namespace torch {
+namespace jit {
+
+using c10::IValue;
+using caffe2::serialize::PyTorchStreamReader;
+using caffe2::serialize::PyTorchStreamWriter;
+using caffe2::serialize::ReadAdapterInterface;
+using torch::jit::TensorIndexMap;
+
+namespace {
+
+void writeArchive(
```

Comment:
can the be reused? i.e. is this (could it be) a utility function of general use to other backport functions.

Reply:
This function is mostly for writing bytecode part and is customized for v5 to v4 change. It adds some updates to bytecode values and write it to archive. I assume for v6 to v5, it needs to update bytecode values in its own way and write it to archive. That's the reason I leave it in `backport_v5_to_v4.cpp`.

Reply:
Maybe rename it is something specific, like writeArchiveV4. No strong preference.

Reply:
Rename in the latest commit